components/ws-proxy/pkg/proxy/routes_test.go

```diff
@@ -56,14 +56,15 @@ var (
 		},
 	}
 
-	ideServerHost       = "localhost:20000"
-	workspacePort       = uint16(20001)
-	supervisorPort      = uint16(20002)
-	workspaceDebugPort  = uint16(20004)
-	supervisorDebugPort = uint16(20005)
-	workspaceHost       = fmt.Sprintf("localhost:%d", workspacePort)
-	portServeHost       = fmt.Sprintf("localhost:%d", workspaces[0].Ports[0].Port)
-	blobServeHost       = "localhost:20003"
+	ideServerHost           = "localhost:20000"
+	workspacePort           = uint16(20001)
+	supervisorPort          = uint16(20002)
+	workspaceDebugPort      = uint16(20004)
+	supervisorDebugPort     = uint16(20005)
+	debugWorkspaceProxyPort = uint16(20006)
+	workspaceHost           = fmt.Sprintf("localhost:%d", workspacePort)
+	portServeHost           = fmt.Sprintf("localhost:%d", workspaces[0].Ports[0].Port)
+	blobServeHost           = "localhost:20003"
```

Comment:
nits: I prefer [net.JoinHostPort](https://pkg.go.dev/net#JoinHostPort) to `fmt.Srintf`

Reply:
thanks for the hint, I will remember it 💡